draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -9830,9 +9837,9 @@ Content-Type: text/plain
 </t>
 <t>
    The definition of a new status code ought to specify whether or not it is
-   cacheable. Note that all status codes can be cached if the response they
+   heuristically cacheable. Note that all status codes can be cached if the response they
```

Comment:
"heuristically cacheable" is the phrasing used in Section 15.1, so this was just an attempt to increase consistency.